packages/yourloops/lib/utils.ts

```diff
@@ -123,3 +124,31 @@ export function getUserInitials(user: IUser): string {
 export function getUserEmail(user: IUser): string {
   return Array.isArray(user.emails) ? user.emails[0] : user.username;
 }
+
+/**
+ * YLP-878 Wrong settings for glucose units uploaded by the handset
+ * @param settings Settings received
+ * @returns Fixed settings
+ */
+export function fixYLP878Settings(settings: Settings | undefined | null): Settings {
+  if (_.isNil(settings)) {
+    return {
+      country: "FR",
+      units: {
+        bg: Units.gram,
+      },
+    };
```

Comment:
why don't we have a default object that we merge with incoming values: wondering if the lodash.merge could make the job to produce the newSettings ?

Reply:
It was to cleanup the settings we used.
To be sure we only have what we want.